system/handlers/formbuilder/item-types/Date.cfc

```diff
@@ -13,13 +13,15 @@ component {
 
 			switch( args.relativeOperator ) {
 				case "lt":
-					args.maxDate = DateAdd( 'd', -1, Now() );
+					var diff = val(args.offset) ? val(-args.offset) : -1;
```

Comment:
They already set default value -1 in 'lt' (same for the 'gt'):
`args.maxDate = DateAdd( 'd', -1, Now() );`
I've replace this line with
`var diff = val(args.offset) ? val(-args.offset) : -1;`